rsciio/edax/_api.py

```diff
@@ -123,7 +129,7 @@ def get_spc_dtype_list(load_all=False, endianess="<", version=0.61):
     """
     Get the data type list for an SPC spectrum.
     Further information about the file format is available `here
-    <https://github.com/hyperspy/hyperspy/files/29506/SPECTRUM-V70.pdf>`__.
+    <https://hyperspy.org/rosettasciio/_downloads/9e2f0ccf5287bb2d17f1b7550e1d626f/SPECTRUM-V70.pdf>`__.
```

Comment:
Do you know how "permanent" is this link, if it can change with new release, etc? Is it possible to use sphinx label as alternative?

Reply:
I changed it to (the new) GitHub links again, which should be "permanent" when not moving the file.

Problem with sphinx label would be that it is readable only in the parsed docstring on read-the-docs and e.g. not in docstrings displayed inline in python.

Reply:
> Problem with sphinx label would be that it is readable only in the parsed docstring on read-the-docs and e.g. not in docstrings displayed inline in python.

Indeed, but this is the same for all sphinx hyperlink? However, this is an issue with rendering and not the docstring itself - hopefully, https://github.com/jupyter/papyri will improve the situation, when it would ready to use!

Reply:
Actually, because this is for the docstring of `get_spc_dtype_list` and not `file_reader`, it doesn't appear in rosettasciio documentation, so it may be better to use with the permanent link from github.

Reply:
> > Problem with sphinx label would be that it is readable only in the parsed docstring on read-the-docs and e.g. not in docstrings displayed inline in python.
> 
> Indeed, but this is the same for all sphinx hyperlink? However, this is an issue with rendering and not the docstring itself - hopefully, https://github.com/jupyter/papyri will improve the situation, when it would ready to use!

But I think, we don't have so many links in the docstrings. Mostly in the user-guide, where using the sphinx references makes sense.